versions-enforcer/pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>versions</artifactId>
+        <groupId>org.codehaus.mojo.versions</groupId>
+        <version>2.14.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>versions-enforcer</artifactId>
+
+    <name>Versions Enforcer</name>
+    <description>Enforcer rules using Versions Maven Plugin</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codehaus.mojo.versions</groupId>
+            <artifactId>versions-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.codehaus.mojo.versions</groupId>
+            <artifactId>versions-test</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.maven.enforcer</groupId>
+            <artifactId>enforcer-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>${mavenVersion}</version>
+            <scope>provided</scope>
```

Comment:
`test` should be ok

Reply:
Unfortunately, not yet 🙂

Still used by `VersionsHelper`, but I will soon remove it.

Reply:
What I will do for the time being is declare a separate (now superfluous) provided dependency to maven-core, and I will simply remove the maven-compat later on.